nixos/modules/services/misc/sonarr.nix

```diff
@@ -27,6 +36,31 @@ in
         '';
       };
 
+      settings = lib.mkOption {
+        inherit (settingsFormat) type;
+        description = "An attribute set containing Sonarr configuration settings.";
+        default = { };
+        example = lib.literalExpression ''
```

Comment:
I think there should be curly brackets in the example, inside the multi-line string, to represent an attrset.

Reply:
I think the example does not even need to use `literalExpression`, and could just directly be an attrset. If we want to add comments or something, _then_ `literalExpression` would be useful.

Reply:
use the attrset directly